src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,171 @@
+All versions of CockroachDB released after November 18, 2024, including patch fixes for prior versions 23.1 onward, are made available under the [CockroachDB Software License][csl]. You can use the dropdown menu at the top of the page to view documentation for other versions.
```

Comment:
A little unclear what "prior versions 23.1 onward" means. Is the suggestion correct?

```suggestion
All versions of CockroachDB released after November 18, 2024, including patch fixes for v23.1 and later, are made available under the [CockroachDB Software License][csl]. You can use the dropdown menu at the top of the page to view documentation for other versions.
```

Reply:
this one got legal touches so even if it's a little unclear i would prefer to leave it be